examples/namespaces_example.cpp

```diff
@@ -0,0 +1,28 @@
+#include <rdf4cpp/rdf.hpp>
+
+#include <iostream>
+
+int main() {
+    using namespace rdf4cpp::rdf;
+
+    Namespace ex("http://example.com/");
+
+    auto ex_A = ex + "A";
+    std::cout << ex_A << std::endl;
```

Comment:
Personal Opinion (yours might be different):

A general suggestion for examples: maybe replace the printing of example values by asserts.
The benefit this has is that users don't have to actually execute the code to see what the expected output is.
At least I personally would never actually bother to execute the example so it would be not helpful to me; but seeing what is does at a glance makes it useful to me.

Example:
```c++
Namespace ex{"http://example.com/"};

auto ex_A = ex + "A";
assert(std::string{ex_A} == "http://example.com/A"); // or something simmilar that shows what the expected output is
```

Reply:
good point